Doc/library/socket.rst

```diff
@@ -149,6 +149,14 @@ created.  Socket addresses are represented as follows:
 
   .. versionadded:: 3.6
 
+- :const:`AF_VSOCK` allows communication between virtual machines and
+  their hosts. The sockets are represented as a ``(CID, port)`` tuple
+  where the context ID or CID and port are integers.
+
+  Availability: Linux >= 4.8. QEMU >= 2.8.
```

Comment:
Given that vsockets originated with VMWare, add a note that they are available in that environment also?

Reply:
I meant availability when running under the VMWare hypervisor. As for the kernel, to keep it simple, whichever version supplies full functionality with both qemu and vmware.